docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,103 @@
+
+Ollama could run the iGPU 780M of AMD Ryzen CPU at Linux base on ROCm. There only has a little extra settings than Radeon dGPU like RX7000 series.
+
+## Keys for usage
+- Ryzen 7000s/8000s CPU with iGPU 780M
+- amdgpu driver and rocm6.0
+- Linux OS is required (Windows and WSL2 are not supported)
+- BIOS must be set to enable the iGPU and dedicate > 1GB RAM to VRAM
+- HSA_OVERRIDE_GFX_VERSION="11.0.0" is set (extral setting for AMD iGPU-780M)
+
+## Prerequisites
+0. Set UMA for iGPU in BIOS. (at least >1GB, recommend to >8GB for Llama3:8b q4_0 model size is 4.7GB)
+1. Install GPU Driver and ROCm
+	Refer to
+	- [AMD ROCm™ documentation — ROCm Documentation](https://rocmdocs.amd.com/en/latest/)
+  	- [AMD ROCm™ Quick start installion](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/quick-start.html#rocm-install-quick)
+
+2. Install Ollama
+	
+ 	*`curl -fsSL https://ollama.com/install.sh | sh`*
+
+## Steps
+The iGPU is not detected by Ollama at default. We need extra steps to enable it.
+1. Stop the ollama.service
+   
+	`sudo systemctl stop ollama.service`
+	
+   Then find out the pid of ollama.service by 'ps -elf | grep ollama' and then 'kill -p [pid]'
+   
+2. for iGPU 780 w/ ROCm ( not work in WSL, need run in Linux)
+
+	`HSA_OVERRIDE_GFX_VERSION="11.0.0" ollama serve &`
+
+3. Run ollama
+   
+   `ollama run tinyllama`
+   
+   Use rocm-smi to watch the utilization of iGPU When run ollama with ROCm.
+
+Another way to replace the step-2 above is to config the ollama.service for iGPU with ROCm as default.
+	
+  `sudo systemctl edit ollama.service`
+
+Add the contents into the /etc/systemd/system/ollama.service.d/override.conf
+
+```
+[Service]
+Environment="HSA_OVERRIDE_GFX_VERSION=11.0.0"
+```
+
+Then Reboot the Linux or just restart the ollama.srevice by,
+	 
+  `sudo system restart ollama.service`
+
+
+### Check iGPU utilizaion
+
+Run `ollama ps`
+
+```
+$ ollama ps
+NAME            ID              SIZE    PROCESSOR       UNTIL
+llama2:latest   78e26419b446    5.6 GB  100% GPU        4 minutes from now
+```
+
+**Examples of iGPU 780M w/ ROCm** 
+```
+$HSA_OVERRIDE_GFX_VERSION="11.0.0" /usr/local/bin/ollama serve &
```

Comment:
I would remove this line - as long as the reader has followed the steps above, the `ollama run` will work and use the iGPU.